### PR TITLE
Feat: Home, Popular Monuments and Monument Details functionality implemented for mobile 

### PR DESCRIPTION
**Pull request by vkprogrammer-001**
This PR implements the following functionality:

- Home Screen with bottom Navigation Bar (for mobile android/ios)
- Popular Monuments
- Monument Details (partially done, as the remaining part involves other features)
- image picking and cropping for adding new posts (needs to implement the feed functionality)
- image carousel

This PR implements the following UIs:
- Home Screen with bottom Navigation Bar
- Popular Monuments Screen
- Monument Details Screen
- image carousel screen

### CODE REVIEW

**Review by mdmohsin7 — APPROVED** [review]
There's a typo in the `pick_image.dart` file. Instead of `pick_image`, it is written as `pick_imge`. This anyway can be fixed when working on the feed functionality.

**Comment by vkprogrammer-001** [comment]
@mdmohsin7 ok, I have fixed it. Thanks

File: lib/presentation/home/mobile/widgets/new_post_bottom_sheet.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        });
+  }
+  Future newPostPickImage(ImageSource source) async {
+    File image = await PickImage.takePicture(imageSource: source);
+    File croppedImage =
+        await PickImage.cropImage(image: image, ratioX: 1, ratioY: 1);
+        //TODO: create a new post
+    // Navigator.of(context).pushNamed(NewPostScreen.route,
+    //     arguments: NewPostScreenArguments(pickedImage: croppedImage));
+    print(croppedImage);
```
Comment (jddeep) [inline lib/presentation/home/mobile/widgets/new_post_bottom_sheet.dart]: please use `log()` instead of `print()`

Comment (vkprogrammer-001) [reply lib/presentation/home/mobile/widgets/new_post_bottom_sheet.dart]: @jddeep done

**Review by jddeep — APPROVED** [review]

**Pull request merged**